### ISSUE TITLE
plist files are not deleted during the upgrade to 0.16.x.

### ISSUE THREAD

**Issue by djallado** [learningequality/installers#282]
**Branch:**  `develop`

**Expected Behavior:** During upgrade, the installer will remove the `org.learningequality.kalite.plist` in root `/Library/Launchagents/`.

**Current Behavior:**  Root `/Library/LaunchAgents/` still contains `org.learningequality.kalite.plist` during the upgrade.

**Steps to reproduce:** 
1. Use fresh install `Yosemite` in virtual machine.
2. Install `0.15.0.dmg`.
3. Launch `KA-Lite-Monitor` from `Applications` folder. 
4. Click `Start KA Lite` from the menu item.
5. Click `Open in Browser` menu item when ready, and Navigate in the browser to check if it is working. 
6. Click `Quit` from the menu item.
7. Install `0.16.0.pkg`.
8. After a successful installation, Launch `KA-Lite` from `Applications` folder. 
9. Click `Start KA Lite` from the menu item.
10. Observed that there's an error in the app.
11. Find `org.learningequality.kalite.plist` from root `/Library/LaunchAgents/`.
![screen shot 2015-11-25 at 10 19 07 am](https://cloud.githubusercontent.com/assets/8664071/11405699/6118acb6-93e4-11e5-9af3-2be3722d752b.png)

`org.learningequality.kalite.plist` is not deleted in root `/Library/LaunchAgents/`.

**Comment by cpauya** [comment]
@djallado I changed the title to "plist files..." because we are actually checking for possible multiple instances of a .plist that could affect our installer/app:

1. /Library/LaunchAgents/org.learningequality.kalite.plist
2. ~/Library/LaunchAgents/org.learningequality.kalite.plist

So during your tests, if these are already deleted by the installer, please comment here so we can close this.

**Comment by cpauya** [comment]
@djallado Let's test and confirm that this is fixed in #309 already.

**Comment by cpauya** [comment]
@arceduardvincent I think @amodia is using `~/Library/LaunchAgents/org.learningequality.kalite.prefs.plist` -- did we also include this on the uninstall script?

**Comment by djallado** [comment]
@cpauya I tested this again and `org.learningequality.kalite.plist` from `/Library/LaunchAgents/` was removed when I upgrade to 0.16.0.

Here's the snapshots:
![screen shot 2015-12-09 at 11 02 43 am](https://cloud.githubusercontent.com/assets/8664071/11675682/996e3798-9e66-11e5-970d-ba06cbb39e88.png)
This is the plist when I install `0.15.0`.

![screen shot 2015-12-09 at 11 11 00 am](https://cloud.githubusercontent.com/assets/8664071/11675685/a24b9ed2-9e66-11e5-8009-c67951d3b471.png)
This the plist when I upgrade to `0.16.0`.
